### PR TITLE
[Code Completion] Suffix ": " instead of "=" when completing attribute argument names

### PR DESCRIPTION
**Pull request by rintaro**
#### What's in this pull request?

All _named_ attribute parameters have been changed to use colon.

---

Before merging this pull request to apple/swift repository:
- [ ] Test pull request on Swift continuous integration.

<details>
  <summary>

Triggering Swift CI</summary>

The swift-ci is triggered by writing a comment on this PR addressed to the GitHub user @swift-ci. Different tests will run depending on the specific comment that you use. The currently available comments are:

**Smoke Testing**

| Platform | Comment |
| --- | --- |
| All supported platforms | @swift-ci Please smoke test |
| All supported platforms | @swift-ci Please smoke test and merge |
| OS X platform | @swift-ci Please smoke test OS X platform |
| Linux platform | @swift-ci Please smoke test Linux platform |

A smoke test on macOS does the following:
1. Builds the compiler incrementally.
2. Builds the standard library only for macOS. Simulator standard libraries and
   device standard libraries are not built.
3. lldb is not built.
4. The test and validation-test targets are run only for macOS. The optimized
   version of these tests are not run.

A smoke test on Linux does the following:
1. Builds the compiler incrementally.
2. Builds the standard library incrementally.
3. lldb is built incrementally.
4. The swift test and validation-test targets are run. The optimized version of these
   tests are not run.
5. lldb is tested.

**Validation Testing**

| Platform | Comment |
| --- | --- |
| All supported platforms | @swift-ci Please test |
| All supported platforms | @swift-ci Please test and merge |
| OS X platform | @swift-ci Please test OS X platform |
| OS X platform | @swift-ci Please benchmark |
| Linux platform | @swift-ci Please test Linux platform |

**Lint Testing**

| Language | Comment |
| --- | --- |
| Python | @swift-ci Please Python lint |

Note: Only members of the Apple organization can trigger swift-ci.
</details>

### CODE REVIEW

File: lib/IDE/CodeCompletionResultBuilder.h [hunk 1]
```diff
@@ -255,7 +255,7 @@ class CodeCompletionResultBuilder {
                      DeclAttrParamKeyword, Name);
     if (NeedSpecify)
       addChunkWithText(CodeCompletionString::Chunk::ChunkKind::
-                       DeclAttrParamEqual, "=");
+                       DeclAttrParamEqual, ": ");
```
Comment (rintaro) [inline lib/IDE/CodeCompletionResultBuilder.h]: Should I rename `DeclAttrParamEqual` => `DeclAttrParamColon` ?

Comment (jrose-apple) [inline lib/IDE/CodeCompletionResultBuilder.h]: cc @benlangmuir

**Comment by jrose-apple** [comment]
@swift-ci Please smoke test

File: lib/IDE/CodeCompletionResultBuilder.h (hunk 1 above)
Comment (benlangmuir) [inline lib/IDE/CodeCompletionResultBuilder.h]: Yes, please!  And please update the doc comment above its declaration in CodeCompletion.h.

**Comment by rintaro** [comment]
Renamed `DeclAttrParamEqual` to `DeclAttrParamColon`
Updated the doc comment.

@swift-ci Please smoke test

**Comment by rintaro** [comment]
macOS: [Build Status](https://ci.swift.org/view/Pull%20Request/job/swift-PR-osx-smoke-test/1214/)
Linux: [Build Status](https://ci.swift.org/job/swift-PR-Linux-smoke-test/748/)

**Comment by benlangmuir** [comment]
@swift-ci Please smoke test

**Comment by benlangmuir** [comment]
For swift-3.0-branch: https://github.com/apple/swift/pull/4325

**Pull request merged**